<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGame</class>
 <widget class="QMainWindow" name="QtGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGame</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Velocity">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocity</string>
    </property>
   </widget>
   <widget class="QLabel" name="Radious">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Radious</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckMark">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>76</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="My_table">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="Velocity_Change">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="Radious_Change">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>42</width>
      <height>0</height>
     </size>
    </property>
    <property name="prefix">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPool_Game">
    <property name="title">
     <string>Pool Game</string>
    </property>
   </widget>
   <addaction name="menuPool_Game"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
